src/views/LoginView.vue

```diff
@@ -288,7 +288,9 @@ export default {
   min-height: 100vh;
   text-align: center;
   color: #FFF4D7;
-  background: center / cover no-repeat  url('img/game/board-background.svg');
+  background-image: url('/img/game/board-background.svg');
+  background-size: cover;
+  background-position: center;
```

Comment:
I have no idea why this should make a difference @seriouslysean @Haviles04... But I deployed this to prod without merging and it works
![image](https://github.com/cuttle-cards/cuttle/assets/6520704/c10dcda5-7d15-4847-ae0c-5a8faf0663eb)


I'm asking in the vue discord to see if anyone can explain why this difference matters

Reply:
Update: needed `url(/img...)` not just `url(img...)` prefix / indicates the domain root